### DISCUSSION TITLE
Water leak sensor with TYWE3S chip

### DISCUSSION THREAD

**Discussion by array81** [arendst/Tasmota#16030] [Q&A] [answered]
I have a Tuya water leak sensor taht use TYWE3S chip.
The sensor is similar to Y09 but not the same.
https://templates.blakadder.com/Y09.html

![180047645-dfe72c29-ff4d-42b7-966b-528382259161](https://user-images.githubusercontent.com/9898768/180078624-c77b5307-f52d-43f0-841e-2263a81595a6.jpg)

Has anyone managed to install Tasmota?

**Comment by array81** [comment]
I'd like try to flash it as explained in the article but "the left" of board is not the same so I don't know GND and RST pin.
![IMG_20220721_222916](https://user-images.githubusercontent.com/9898768/180310062-79b0f1f7-6f9b-4f81-bfab-329bc3051c77.jpg)

**Reply by sfromis** [reply]
GND is easy to find on the TYWE3S.
https://tasmota.github.io/docs/Pinouts/#tywe3s

While you may indeed have to disable the other MCU chip, you could try without. To find where the RST pin is on that chip, check the data sheet. Your image shows nothing about what chip it is.

**Comment by array81** [comment, marked as answer]
I managed to flash Tasmota.
The instructions of the first link are fine but you must connect the second (where there is a minus sign) and the third pin on the opposite side of the TYWE3S.
Hope it helps.
